openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx

```diff
@@ -332,6 +335,35 @@ const DashboardDetails = ({
       tagUpdateHandler(updatedDashboard);
     }
   };
+
+  const handleRestoreDashboard = async () => {
+    const data: CreateDashboard = {
+      charts: dashboardDetails.charts,
+      dashboardUrl: dashboardDetails.dashboardUrl,
+      description: dashboardDetails.description,
+      displayName: dashboardDetails.displayName,
+      extension: dashboardDetails.extension,
+      name: dashboardDetails.name,
+      owner: dashboardDetails.owner,
+      service: dashboardDetails.service,
+      tags: dashboardDetails.tags,
+    };
```

Comment:
```suggestion
    const data: CreateDashboard = cloneDeep(dashboardDetails);
```

Reply:
@chirag-madlani  `PUT` request is accepting only some required value to restore, by this method we are sending the entire value of `GET` which is not accepted.

<img width="715" alt="image" src="https://user-images.githubusercontent.com/66266464/197338448-8b8b74e0-c23d-486b-a70b-eddf63c6576b.png">